sidebars.ts

```diff
@@ -662,48 +663,6 @@ const sidebars: SidebarsConfig = {
               ],
             },
             "product-analytics/dashboards",
-            {
```

Comment:
this is a lot of pages to orphan, probably need to redirect these?

Reply:
These aren't deleted - moved up to line ~400